coretex/cli/project.py

```diff
@@ -0,0 +1,54 @@
+from typing import Optional
+
+import click
+from ..configuration import loadConfig, saveConfig
+from .. import Project, ProjectType
+
+
+def createProject(name: str) -> None:
+    try:
+        new_project = Project.createProject(name, ProjectType.other)
+        if new_project is not None:
+            config = loadConfig()
+            config["projectId"] = new_project.id
+            saveConfig(config)
+    except Exception as e:
+        raise click.ClickException(f"Failed to create project: {e}")
+
+
+@click.command()
+@click.option("--name", "-n", type = str, help = "Project name")
+@click.option("--id", type = int, help = "Project ID")
+def select(name: Optional[str], id: Optional[int]) -> None:
+    if name is None and id is None:
+        raise click.UsageError("Please use either --name / -n (for project name) or --id (for project ID)")
+
+    if name:
+        click.echo("Checking project name")
+        projects = Project.fetchAll(name = name, include_sessions = 1)
```

Comment:
Use `Project.fetchOne` instead of `Project.fetchAll`. What is `include_sessions` parameter?